src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -330,6 +346,24 @@ export var analyticsActionCauseList = {
     name: 'breadcrumbFacet',
     type: 'breadcrumb'
   },
+  /**
+   * The search-as-you-type event that gets logged when a breadcrumb facet is selected and the query is updated.
+   *
+   * `actionCause`: `'breadcrumbMLFacet'`
+   * `actionType`: `'breadcrumb'`
+   *
+   * Logging an event with this actionType also adds the following key-value pair in the custom data property of the Usage Analytics HTTP service request.
```

Comment:
key-value pair -> key-value pairs

Reply:
This comment also applies to the `mlFacetSelect`, `mlFacetDeselect`, and `mlFacetClearAll` documentation